spec/routes/status_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Citygram::Routes::Status do
+  include Citygram::Routes::TestHelpers
+
+  describe 'GET /status' do
+    it 'returns a non-empty body' do
+        stub_request(:get, "https://api.twilio.com/2010-04-01").
```

Comment:
Refactored to utilize twilio url from env variables.

This has been demo'd in a development environment, but not a "demo" environment at this point.

"GET /status HTTP/1.1" 200

**Here is sample output:**

{"status":"ok","dependencies":["postgres","twilio"],"updated":1438268117}